Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -3699,7 +3701,7 @@ private ElementInfo GetRootElement()
         {
             ElementInfo RootEI = new ElementInfo();
             RootEI.ElementTitle = "html";
-            RootEI.ElementType = "root";
+            RootEI.ElementTypeEnum = eElementType.Root;
             RootEI.Value = string.Empty;
```

Comment:
this need to revert to RootEI.ElementType = "root";, why we have eElementType.Root? Root is not ElmentType so no need the Enum here